application/tox.ini

```diff
@@ -0,0 +1,17 @@
+[tox]
```

Comment:
It is kind of confusing to have two seperate tox.ini/setup.pys, one at the root and one in this application directory.

I would recommend either:

1. Consolidating everything into one bundle.
2. Putting the python-libprio bits into a subdirectory off the root.

Likely (2) makes the most sense at this point.

Reply:
I agree, but I think this should be left until #59 is merged in to avoid excess merge conflict. This PR is important for python multiprocessing and significantly changes how the wrapper library should be used.

Reply:
Ok, as long as there is a plan!